src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Enums.cs

```diff
@@ -8,150 +8,150 @@ namespace System.Runtime.Intrinsics.Arm
     public enum SveMaskPattern : byte
     {
         /// <summary>
-        /// POW2
+        ///   <para>POW2</para>
```

Comment:
I don't think it's needed for single lines, but might be OK anyway.

Reply:
In this case it was more about getting the bulk of it correct without needing to manually audit all the lines.

I'll see if I can recognize the single line case and fix it up to not include the tag...